yarn.lock

```diff
@@ -909,6 +1344,357 @@
   dependencies:
     "@sinonjs/commons" "^1.7.0"
 
+"@smithy/abort-controller@^2.0.12":
```

Comment:
Are oll of those entries dependencies from aws?

Reply:
Not sure. I have reverted the upstream.